<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogElitePMC</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="SessionSystemVisits">
     <property name="text">
      <string>Session System Visits 100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="SystemsVisited">
     <property name="text">
      <string>Systems Visited 1,000,000</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Star System to clipboard</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
